examples/JsonSerialization/Program.cs

```diff
@@ -163,6 +162,7 @@ await producer
             {
                 // Note: a subject name strategy was not configured, so the default "Topic" was used.
                 var schema = await schemaRegistry.GetLatestSchemaAsync(SubjectNameStrategy.Topic.ConstructValueSubjectName(topicName));
+                Console.WriteLine("schema id : " + schema.Id);
```

Comment:
how does this look? capitalization you've used is different than the next line, so inconsistent, also the next WriteLine has a preceding \n. maybe either get rid of this line, or incorporate the schema id into the next.

Reply:
maybe: `Console.WriteLine($"\nThe JSON schema (id: {schema.Id}) corresponding to the written data:");`

Reply:
and remove the above line.

Reply:
Getting rid of this line.